HISTORY.md

```diff
@@ -787,9 +787,9 @@
 
 ## 14.0.68 alpha 2020-05-06
 
-* fix(common): enable build for win x64, use global VERSION.md and fix decxstr() bug (#3076)
+* fix(common/core/desktop): enable build for win x64, use global VERSION.md and fix decxstr() bug (#3076)
 * change(android): Add methods to go between LanguageResource and JSON (#3079)
-* feat(common): auto-update for package versioning (#3078)
+* feat(common/resources): auto-update for package versioning (#3078)
```

Comment:
This one's debatable, but I think the original scope is more accurate.  It's not a resource-targeting script - it actually has direct effects on our final build products.

Reply:
reverted to (common)